src/Core/MailTemplate/FolderThemeScanner.php

```diff
@@ -45,6 +45,19 @@ final class FolderThemeScanner
      */
     private $baseThemeFolder;
 
+    /**
+     * @var string
+     */
+    private $moduleDirectory;
+
+    /**
+     * @param string $moduleDirectory
+     */
+    public function __construct($moduleDirectory)
```

Comment:
```suggestion
    public function __construct(string $moduleDirectory)
```

Reply:
I wasn't sure if we could use it, thank you